frontend/apps/marketing/src/components/actionBlocks/fullWidthActionBlock/FullWidthActionBlock.tsx

```diff
@@ -19,6 +21,7 @@ export type FullWidthActionBlockContentfulProps = Omit<
   primaryButton: LinkEntry;
   secondaryButton: LinkEntry;
   background: EntryFields.Text;
+  publishedDate: EntryFields.Date | undefined;
```

Comment:
nit: 
Isn't it possible to simplify this to this? :

```suggestion
  publishedDate?: EntryFields.Date;
```